src/meta/src/manager/catalog/mod.rs

```diff
@@ -162,6 +165,120 @@ where
         Ok(())
     }
 
+    /// `temp_drop_builtin_schema` is only used to ensure backward compatibility in the early
+    /// version.
+    async fn temp_drop_builtin_schema(&self) -> MetaResult<()> {
+        let core = &mut *self.core.lock().await;
+        let database_core = &mut core.database;
+        let user_core = &mut core.user;
+        let to_drop_schema = database_core
+            .schemas
+            .iter()
+            .filter(|(&id, schema)| {
+                is_system_schema(&schema.name) && database_core.schema_is_empty(id)
```

Comment:
Since in early version we don't reject creating relations inside system catalog schemas: `rw_catalog` and `information_schema`, so we only drop empty builtin schemas here and let the frontend compatibility handle it. It's a little bit dirty and I have no better solutions. 🥵

Reply:
Actually even though in early version user can create tables/mvs in that system schemas and do dml operations, they still cannot query on it. Because in the frontend only the system tables will be bound under the system schemas.

Reply:
Or we can move all the user created relations to `public` to achieve more compatibility and let frontend do nothing. 🥵  @xiangjinwu WDUT?